mage/query-modules/python/conditional-execution.md

```diff
@@ -0,0 +1,177 @@
+---
+id: conditional-execution
+title: conditional_execution
+sidebar_label: conditional_execution
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+  <span
+    style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+    {children}
+  </span>
+);
+
+Your queries might require conditional execution logic that can’t be adequately
+expressed in Cypher. The `do` module makes it possible to define complex logic
+and use it to control query execution.
+
+[![docs-source](https://img.shields.io/badge/source-conditional_execution-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/python/do.py)
+
+| Trait              | Value                                                 |
+| ------------------ | ----------------------------------------------------- |
+| **Module type**    | <Highlight color="#FB6E00">**module**</Highlight>     |
+| **Implementation** | <Highlight color="#FB6E00">**Python**</Highlight>     |
+| **Parallelism**    | <Highlight color="#FB6E00">**sequential**</Highlight> |
+
+## Procedures
+
+:::info
+
+Using the following procedures to run queries that execute global operations is
+currently not supported and returns a warning.
+The operations in question are:
+
+* index creation/deletion
+* constraint creation/deletion
+* changing the isolation level globally
+* setting the storage mode
+
+:::
+
+### `case(conditionals, else_query, params)`
+
+Given a list of condition-query pairs, `do.case` executes the query associated
+with the first condition evaluating to `true` (or the `else query` if none are
```

Comment:
```suggestion
with the first condition evaluating to `true` (or the `else query` if none are
```
This sounds strange... "the first condition evaluating TO `true`
What do you mean by this? "associated with the first condition that is/results in `true` " can we just say this?